ui/packages/catalog-ui-search/src/main/webapp/behaviors/selection.behavior.js

```diff
@@ -42,13 +42,15 @@ Behaviors.addBehavior(
   Marionette.Behavior.extend({
     events() {
       return {
+        [`click ${this.options.noClick}`]: 'handleCheckboxClick',
```

Comment:
✏️ my fault, but thinking `noClick` should be renamed something with checkbox to match functions/variables in this class